.travis.yml

```diff
@@ -66,6 +67,16 @@ matrix:
     - rvm: 1.8.7
       env: PUPPET_GEM_VERSION="~> 4.8.0"
     - rvm: 1.8.7
+      env: PUPPET_GEM_VERSION="~> 4.9.0"
+    - rvm: 1.9.3
+      env: PUPPET_GEM_VERSION="~> 4.9.0"
+    - rvm: 2.0.0
+      env: PUPPET_GEM_VERSION="~> 4.9.0"
+    - rvm: 1.8.7
+      env: PUPPET_GEM_VERSION="~> 4"
+    - rvm: 1.9.3
```

Comment:
Support for Ruby < 2.1.0 got deprecated with Puppet 4.9 and tests will fail otherwise.

https://docs.puppet.com/puppet/latest/release_notes.html#ruby-20-series